pkg/rancher-desktop/main/commandServer/settingsValidator.ts

```diff
@@ -241,7 +241,7 @@ export default class SettingsValidator {
         } else {
           // newSettings[k] should be valid JSON because it came from `JSON.parse(incoming-payload)`.
           // It's an internal error (HTTP Status 500) if it isn't.
-          errors.push(`Setting ${ fqname } should be a simple value, but got <${ JSON.stringify(newSettings[k]) }>.`);
+          errors.push(`Setting "${ fqname }" should be a simple value, but got "${ JSON.stringify(newSettings[k]) }".`);
```

Comment:
There are a number of lines in this file (and possibly in other files) that used `<...>` as "quotes" for a variable. Here the style is changed to `"..."` but in most other places it was left as `<...>`.

It doesn't matter to me either way, but we should be consistent with this: all variables should either be switched to the `"..."` quoting style, or left as the `<...>` quoting style.

Reply:
I want to drop `<...>` as a quoting form, and try to use just double-quotes whenever possible.